### ISSUE TITLE
spotify: can't install

### ISSUE THREAD

**Issue by krey** [NixOS/nixpkgs#36991]
## Issue description

```
error: cannot download spotify-client_1.0.69.336.g7edcc575-39_amd64.deb from any mirror
builder for ‘/nix/store/vzplq6hkqlp4sq6kwa5dls1skpalqsy3-spotify-client_1.0.69.336.g7edcc575-39_amd64.deb.drv’ failed with exit code 1
```

Commit that seems have to broken things:
https://github.com/NixOS/nixpkgs/commit/b7569bfb386607d41305c9b497a2d4063e921e5e

The 1.0.69 deb was removed from https://repository-origin.spotify.com/pool/non-free/s/spotify-client/

@globin 

### Steps to reproduce

```
nix-env -i spotify
```

## Technical details

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.78, NixOS, 17.09.3159.c665fcca9e7 (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(root): `"nixos-17.09.3159.c665fcca9e7, nixos-unstable-18.03pre130932.cc4677c36ee"`
 - channels(krey): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`
```

**Comment by coretemp** [comment]
I could reproduce this via the faster `nix-env -iA nixos.spotify` in about 10 seconds.

**Comment by flokli** [comment]
@Mic92 This can be closed, as #37273 got merged.
